src/components/HeaderMenu/HeaderMenu.js

```diff
@@ -1,18 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { settings } from 'carbon-components';
+import FocusTrap from 'focus-trap-react';
 
 const { prefix } = settings;
 
 function HeaderMenu({ children }) {
   return (
-    <div className={`${prefix}--bmrg-header-drop-down`}>
-      <ul className={`${prefix}--bmrg-header-drop-down__items`}>
-        {React.Children.map(children, (child) => (
-          <li className={`${prefix}--bmrg-header-drop-down__item`}>{child}</li>
-        ))}
-      </ul>
-    </div>
+    <FocusTrap active focusTrapOptions={{ allowOutsideClick: true }}>
```

Comment:
Focus trap here to keep the tabbing inside the menu while it is open. ESC closes it.